bedrock/firefox/templates/firefox/includes/firefox-footer-links.html

```diff
@@ -27,7 +27,7 @@
 
 {% if show_desktop %}
   <ul class="fx-footer-links os_desktop os_other">
-    <li><a href="{{ firefox_url('desktop', 'all', channel) }}">{{ _('Firefox for Other Platforms & Languages') }}</a></li>
-    <li><a href="{{ firefox_url('desktop', 'notes', channel) }}">{{ _('Release Notes') }}</a></li>
+    <li><a id="fx-footer-links-desktop-all" href="{{ firefox_url('desktop', 'all', channel) }}">{{ _('Firefox for Other Platforms & Languages') }}</a></li>
+    <li><a id="fx-footer-links-release-notes" href="{{ firefox_url('desktop', 'notes', channel) }}">{{ _('Release Notes') }}</a></li>
```

Comment:
It is not. I added this last minute to be consistent with its sibling, which I'm still not sure is the "right" choice. Happy to remove the unused id.